garage/experiment/local_tf_runner.py

```diff
@@ -9,7 +9,7 @@
 import tensorflow as tf
 
 from garage.misc.logger import logger
-from garage.sampler.parallel_sampler import singleton_pool
+from garage.sampler.stateful_pool import singleton_pool
```

Comment:
rebase?

Reply:
I talked to Keren and this is a bug. It wasn't detected in CI.

Reply:
yeah i just hit this myself